scipy/integrate/_quad_vec.py

```diff
@@ -140,6 +141,8 @@ def quad_vec(f, a, b, epsabs=1e-200, epsrel=1e-8, norm='2', cache_size=100e6, li
         Default: 'gk21' for finite intervals and 'gk15' for (semi-)infinite
     full_output : bool, optional
         Return an additional ``info`` dictionary.
+    args : tuple, optional
+        Extra arguments to pass to function, if any.
 
     Returns
```

Comment:
```suggestion
    args : tuple, optional
        Extra arguments to pass to function, if any.

        .. versionadded:: 1.8.0

    Returns
```

Reply:
Once merged, this will need a release note here https://github.com/scipy/scipy/wiki/Release-note-entries-for-SciPy-1.8.0

Reply:
Cool! I'll that after this lands.